doc/nrf/protocols/matter/overview/group_communication.rst

```diff
@@ -0,0 +1,259 @@
+            epochKey2 = c0c1c2c3c4c5c6c7c8c9cacbcccdcecf
+            epochStartTime2 = 1110002
+
+* To see the groups, use the following command:
+
+  .. code-block:: none
+
+     ./chip-tool groupsettings show-groups
+     ...
+       +-------------------------------------------------------------------------------------+
+       | Available Groups :                                                                  |
+       +-------------------------------------------------------------------------------------+
+       | Group Id   |  KeySet Id     |   Group Name                                          |
+       | 0x101           0x1a1            Group #1                                           |
+       | 0x102           0x1a2            Group #2                                           |
+       | 0x103           0x1a3            Group #3                                           |
+       +-------------------------------------------------------------------------------------+
+
+Group Communication triggered from the controller
+=================================================
+
+.. tabs::
+
+   .. group-tab:: Nodes
+
+      You need the `chip-tool`_ and the :ref:`matter_light_bulb_sample` sample to run through this use case.
+
+   .. group-tab:: Precondition
+
+      Commission the Light Bulb to the same fabric as the chip-tool.
+
+   .. group-tab:: Steps
+
+      Complete the following steps to create a custom group based on a custom keyset and start Group Communication:
+
+      1. Generate a custom keyset::
+
+         ./chip-tool groupsettings add-keysets 0xabcd 0 3330000 hex:0123456789abcdef0123456789abcdef
+         ./chip-tool groupsettings bind-keyset 0xdcba 0xabcd
+
+      #. Set the keyset on the Light Bulb based on generated controller's group settings::
+
+         ./chip-tool groupkeymanagement key-set-write '{"groupKeySetID":"0xabcd","groupKeySecurityPolicy":0,"epochKey0":"0123456789abcdef0123456789abcdef","epochStartTime0":"3330000","epochKey1":"0123456789abcdef0123456789abcdee","epochStartTime1":"3330001","epochKey2":"0123456789abcdef0123456789abcded","epochStartTime2":"3330002"}' 1 0
+
+      #. Set the keyset to the group map::
+
+         ./chip-tool groupkeymanagement write group-key-map '[{"groupId":"0xdcba","groupKeySetID":"0xabcd","fabricIndex":"1"}]' 1 0
+
+      #. Add the group on the application endpoint (endpoint 1 includes the on/off cluster)::
+
+         ./chip-tool groups add-group 0xdcba Custom_Group 1 1
+
+      #. Set the ACL to permit the Group Communication from a specific group ID::
+
+         ./chip-tool accesscontrol write acl '[{"fabricIndex":"1","privilege":"5","authMode":"2","subjects":["112233"],"targets":null},{"fabricIndex":"1","privilege":"3","authMode":"3","subjects":["0xdcba"],"targets":[{"cluster":null,"endpoint":"1","deviceType":null}]}]' 1 0
+
+      #. Send a command to toggle on/off on the application endpoint (endpoint 1 includes the onoff cluster)::
```

Comment:
onoff->on/off? I think we used on/off convention in all other places.

Reply:
ooh once is on/off and once onoff

Reply:
Fixed.